README.md

```diff
@@ -3,45 +3,92 @@
 
 # What is this?
 
-Load BIOM tables and sample metadata from a lot of studies into Redis to facilitate search using observation IDs or metadata.
+Load BIOM tables and sample metadata from a lot of studies into Redis to facilitate search using observation IDs or metadata. redbiom is intended as a caching layer. The command line application, for many of the commands, supports Unix pipes allowing for a fair bit of composability. As a hypothetical example, let's say we had loaded a bunch of microbiome data picked againts Greengenes 13_8 at 97% (and into an appropriately named context). To obtain a BIOM table representing all samples in which either OTUs 367523 or 3064251 exist, you can perform the following set of querys:
 
-It is safe to say the interfaces are not stable.
+    $ redbiom search observations --context "greengenes-13_8-97" 367523 3064251 | redbiom fetch samples --from - --output result.biom
+
+Additionally, assuming a comprehensive set of metadata are available, you can obtain summarized environment information:
+    
+    $ redbiom search observations --context "greengenes-13_8-97" 367523 3064251 | redbiom summarize samples --from - --category "env_biome"
```

Comment:
There is a boilerplate replacement that can be done which I'll add. It's somewhat outside of click so it's could bite us later but perhaps it wont